js/modules/k6/http/request_test.go

```diff
@@ -181,7 +181,7 @@ func TestRequestAndBatch(t *testing.T) {
 			}
 
 			assert.Equal(t, 10, reqsCount)
-			assertRequestMetricsEmitted(t, bufSamples, "GET", sr("HTTPBIN_URL/get"), sr("HTTPBIN_URL/redirect/9"), 200, "")
+			assertRequestMetricsEmitted(t, bufSamples, "GET", sr("HTTPBIN_URL/get"), sr("HTTPBIN_URL/get"), 200, "")
```

Comment:
I don't understand the question, sorry.

Reply:
I don't think 
```golang
 			assertRequestMetricsEmitted(t, bufSamples, "GET", url, "", 200, "") 
```
should mean 
```golang
 			assertRequestMetricsEmitted(t, bufSamples, "GET", url, url, 200, "") 
```
I also do think that it will be nice if ... I dunno this is 
```golang
 			assertRequestMetricsEmitted(t, bufSamples, expectmetric{
                             method: "GET", 
                             url: url, 
                             name:"",
                             status: 200,
                             whateverThisIs: "",
                        }) 
```
For this PR maybe just write the `name` for the other tests, instead of leaving it `""`

Reply:
Ahhh, ok, yeah, `url, ""` is confusing, :+1: :+1: for `url, url,`... A full refactoring of the test can wait for a future PR though :smile:

Reply:
I added the redirect checks in 70144e8f, and while I agree that the `name = url` override is unnecessary, I'd rather leave that change for a follow-up PR.

Reply:
Changed this to check all redirects in e6b65c87.